huaweicloud/services/cts/resource_huaweicloud_cts_data_tracker.go

```diff
@@ -283,46 +285,58 @@ func resourceCTSDataTrackerRead(_ context.Context, d *schema.ResourceData, meta
 	allTrackers := *response.Trackers
 	ctsTracker := allTrackers[0]
 
-	d.Set("region", region)
-	d.Set("name", ctsTracker.TrackerName)
-	d.Set("lts_enabled", ctsTracker.Lts.IsLtsEnabled)
-	d.Set("validate_file", ctsTracker.IsSupportValidate)
+	mErr = multierror.Append(
+		mErr,
```

Comment:
the following way makes it easier.

```suggestion
	mErr := multierror.Append(nil,
```